pkg/apis/build/v1alpha2/image_types.go

```diff
@@ -44,7 +44,7 @@ type ImageSpec struct {
 	Builder                  corev1.ObjectReference `json:"builder,omitempty"`
```

Comment:
Should we update the Image status to include the lastCacheImage so that the cache is not lost when the last build failed?

Reply:
Can you explain it a bit more? The cache image will still be there, right? But not updated.